src/IntelliTect.Coalesce/TypeDefinition/Reflection/ReflectionTypeViewModel.cs

```diff
@@ -138,7 +139,7 @@ private static string GetFriendlyTypeName(Type type)
 
             if (!type.IsGenericType)
             {
-                return type.FullName;
+                return type.FullName!;
```

Comment:
There are actually more conditions than just generic type where `FullName` may return null. 
```suggestion
                return type.FullName ?? "";
```